java-squid/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -35,13 +36,19 @@ public void init(){
   public ProgramState checkPreStatement(CheckerContext context, Tree syntaxNode) {
     return context.getState();
   }
-  public void checkPostStatement(CheckerContext context, Tree syntaxNode) {
-    //Default transition
+
+  public ProgramState checkPostStatement(CheckerContext context, Tree syntaxNode) {
+    // Default transition
     context.addTransition(context.getState());
+    return context.getState();
   }
 
   public void checkEndOfExecution(CheckerContext context) {
     // By default do nothing
   }
 
+  public void checkEndOfExecutionPath(CheckerContext context, ConstraintManager constraintManager) {
+    // TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)